srcpkgs/devilutionX/template

```diff
@@ -1,20 +1,61 @@
 # Template file for 'devilutionX'
 pkgname=devilutionX
-version=1.4.1
-revision=3
+version=1.5.2
+revision=1
+_assets_version=4
+_simpleini_version=4.22
+_sdl_audiolib_hash=cc1bb6af8d4cf5e200259072bde1edd1c8c5137e
+_libsmackerdec_hash=91e732bb6953489077430572f43fc802bf2c75b2
+_libmpq_hash=b78d66c6fee6a501cc9b95d8556a129c68841b05
 build_style=cmake
 configure_args="-DVERSION_NUM=$version -DDISABLE_ZERO_TIER=ON"
 hostmakedepends="pkg-config"
 makedepends="SDL2-devel SDL2_image-devel bzip2-devel libsodium-devel
  gtest-devel fmt-devel zlib-devel"
 short_desc="Diablo I engine for modern operating systems"
 maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
-license="Unlicense"
+license="custom:Sustainable Use License"
 homepage="https://github.com/diasurgical/devilutionX"
 changelog="https://raw.githubusercontent.com/diasurgical/devilutionX/master/docs/CHANGELOG.md"
-distfiles="https://github.com/diasurgical/devilutionX/releases/download/${version}/devilutionx-src.tar.xz"
-checksum=f80a5414bb7b5a5ae9f6dbc69cec4ae080c29dd0a8a553cedd405d631011da9f
+distfiles="https://github.com/diasurgical/devilutionX/releases/download/${version}/devilutionx-src.tar.xz
+ https://github.com/brofield/simpleini/archive/refs/tags/v${_simpleini_version}.tar.gz
+ https://github.com/realnc/SDL_audiolib/archive/${_sdl_audiolib_hash}.tar.gz
+ https://github.com/diasurgical/libsmackerdec/archive/${_libsmackerdec_hash}.tar.gz
+ https://github.com/diasurgical/libmpq/archive/${_libmpq_hash}.tar.gz
+ https://github.com/diasurgical/devilutionx-assets/releases/download/v${_assets_version}/spawn.mpq"
+checksum="27dc99cb8b40cb6fccbb5935aac7ce0c906664cc3815bcf9e27195747db46ed3
+ b3a4b8f9e03aabd491aa55fd57457115857b9b9c7ecf4abf7ff035ca9d026eb8
+ 5adc71bee3506b16c505c1662da73fcfdfd878914645784f80fc70db979533a5
+ e565e37ef1ae4f8846d9c9c24b861bc56fdce2d7a4ece47de448e00aa9048b87
+ 348cd9c2be9c067dfcb8a2d6cd6f947ade5088e1543c1e5db1fdc5b12db6aeea
+ 64427cd7c1ba904eaa2e0031c16a6b136d0ecef9abc888c5ff8344b459356e38"
+skip_extraction="v${_simpleini_version}.tar.gz
+ ${_sdl_audiolib_hash}.tar.gz
+ ${_libsmackerdec_hash}.tar.gz
+ ${_libmpq_hash}.tar.gz
+ spawn.mpq"
+repository=nonfree
+restricted=yes
+
+# NOTE: DevilutionX's 3rdParty directory contains a lot of CMake package
+# download instructions that we patch away (even those that are not used) to
+# avoid accidental downloads during the build. The packages that are really
+# needed are provide through this template. Make sure the versions / hashes
+# are what DevilutionX expects and that no new downloads are added. It's a
+# good idea to keep an eye on the build logs.
```

Comment:
does xbps-src's cmake not just fail if it encounters this?

Reply:
No and we have discussed this in IRC a while back. You said there were plans to implement offline-only builds some time ago.